voter-dapp/src/RetrieveRewards.js

```diff
@@ -0,0 +1,201 @@
+import React, { useMemo, useState } from "react";
+import { drizzleReactHooks } from "drizzle-react";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+
+import { useTableStyles } from "./Styles.js";
+import { MAX_UINT_VAL } from "./common/Constants.js";
+
+// Max integer that can be safely stored in a vanilla js int.
+const MAX_SAFE_INT = 2147483647;
```

Comment:
Why not move this to `common/Constants`?

Reply:
Will do.

Reply:
Done.